testpilot/frontend/static-src/styles/_layout.scss

```diff
@@ -85,3 +85,13 @@
 .overflow-hidden {
   overflow: hidden;
 }
+
+.show-on-small {
+  @include respond-to('not-small') {
+    display: none;
+  }
+
+  @include respond-to('small') {
```

Comment:
Careful where you use this, though. We have a bunch of elements that want to be `flex` and not `block`.